### PR TITLE
fix(DB/Gameobject): Fix Silithyst Geyser click invalid

### PR DESCRIPTION
**Pull request by disgrace2013**
Fix Silithyst Geyser click invalid.

## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

## Issues Addressed:

- Closes 

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [x] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [x] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [x] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [ ] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by pangolp** [comment]
Beyond the line break, which I already mentioned in another message, I think we have a problem. Inside the folder:

```
azerothcore/data/sql/updates/pending_db_world/
|-- create_sql.sh
```

You have that script, what it does is generate an SQL file, and where you must copy the code. I don't know if it works the way you did. I always generate the file with the script.

To correct the error, you would have to create the file with the script, copy the content of one file into another, delete the one you uploaded and take advantage of the line break. Then upload the commit, and see if it passes the tests. I hope I can help you with the explanation and that you can do it. If you need help, let me know.

**Comment by pangolp** [comment]
@disgrace2013 You forgot to leave an empty line at the end of the document.

**Comment by pangolp** [comment]
Remember that the file had to be generated with the script, which is located inside the folder. Beyond that, and you can leave a line at the beginning of the document as well as I show you in the image, the truth is that I don't know why the tests fail, they shouldn't.

image

I can't think of another cause, but I generated these changes and we continue to see what the fault could be. Remember to generate the document with the folder script and inside it, add the information.

**Comment by disgrace2013** [comment]
replace the file

**Comment by pangolp** [comment]
Add me to the discord stevej#5879 and I'll see if I can help you.

**Comment by disgrace2013** [comment]
I don't know how to replace the file, this is my first PR .

**Comment by pangolp** [comment]
> I don't know how to replace the file, this is my first PR .

Don't worry, I noticed.
That's why I invited you to write to me on discord.
It happened to all of us the first time.

File: data/sql/updates/pending_db_world/Silithyst-Geyser-fix.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+--
+
```
Comment (elthehablo) [inline data/sql/updates/pending_db_world/Silithyst-Geyser-fix.sql:2]:
```suggestion
```

File: data/sql/updates/pending_db_world/Silithyst-Geyser-fix.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+--
+
+UPDATE `gameobject_template` SET `AIName` = 'SmartGameObjectAI' WHERE `entry` = 181598;
+
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 181598) AND (`source_type` = 1) AND (`id` IN (0));
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `event_param6`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(181598, 1, 0, 0, 64, 0, 100, 0, 0, 0, 0, 0, 0, 0, 75, 29519, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Silithyst Geyser - Add aura Silithyst');
+
+
```
Comment (elthehablo) [inline data/sql/updates/pending_db_world/Silithyst-Geyser-fix.sql:9]:
```suggestion
```

**Review by elthehablo — CHANGES_REQUESTED** [review]
Remove whitelines

**Comment by pangolp** [comment]
Beyond those changes, which I was going to suggest as well, they don't pass the tests, it's strange. If I could avoid the file, I would do it myself, but it won't let me. Although the idea is to help you so you can learn. We have all had problems the first time we wanted to collaborate.

**Comment by pangolp** [comment]
Why did you close the pull request? What happened? Don't give up, come on, it's possible! The first time it's hard to understand the mechanics, but then it becomes very repetitive. Plus, we are here to help you.

**Comment by disgrace2013** [comment]
> Why did you close the pull request? What happened? Don't give up, come on, it's possible! The first time it's hard to understand the mechanics, but then it becomes very repetitive. Plus, we are here to help you.

I have openned a new PR and it was be checked Successfull.

**Pull request closed without merging**